src/fillTank.test.js

```diff
@@ -1,11 +1,147 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
   it('should ', () => {
```

Comment:
The test description is incomplete. Consider providing a meaningful description for this test case.